Google Cloud/Google Cloud Audit/tests/2sv_disable.json

```diff
@@ -9,16 +9,7 @@
   },
   "expected": {
-    "event": {
-      "start": "2022-06-03T12:09:42.501046130Z",
-      "kind": "event",
-      "category": "network",
-      "type": [
-        "info"
-      ],
-      "outcome": "success",
-      "end": "2022-06-03T12:09:42.768509812Z"
-    },
+    "message": "{\n  \"protoPayload\": {\n    \"@type\": \"type.googleapis.com/google.cloud.audit.AuditLog\",\n    \"authenticationInfo\": {\n      \"principalEmail\": \"test-user@example.com\"\n    },\n    \"requestMetadata\": {\n      \"callerIp\": \"203.0.113.255\",\n      \"requestAttributes\": {},\n      \"destinationAttributes\": {}\n    },\n    \"serviceName\": \"login.googleapis.com\",\n    \"methodName\": \"google.login.LoginService.2svDisable\",\n    \"resourceName\": \"organizations/123\",\n    \"metadata\": {\n      \"activityId\": {\n        \"uniqQualifier\": \"-7789616625639281959\",\n        \"timeUsec\": \"1632459962686000\"\n      },\n      \"event\": [\n        {\n          \"status\": {\n            \"success\": true\n          },\n          \"parameter\": [\n            {\n              \"type\": \"TYPE_STRING\",\n              \"label\": \"LABEL_OPTIONAL\",\n              \"value\": \"INfDlrzP9IH8_QE\",\n              \"name\": \"dusi\"\n            }\n          ],\n          \"eventName\": \"2sv_disable\",\n          \"eventType\": \"2sv_change\"\n        }\n      ],\n      \"@type\": \"type.googleapis.com/ccc_hosted_reporting.ActivityProto\"\n    }\n  },\n  \"insertId\": \"-tn3jrd3lko\",\n  \"resource\": {\n    \"type\": \"audited_resource\",\n    \"labels\": {\n      \"service\": \"login.googleapis.com\",\n      \"method\": \"google.login.LoginService.2svDisable\"\n    }\n  },\n  \"timestamp\": \"2021-09-24T05:06:02.686Z\",\n  \"severity\": \"NOTICE\",\n  \"logName\": \"organizations/123/logs/cloudaudit.googleapis.com%2Fdata_access\",\n  \"receiveTimestamp\": \"2021-09-24T05:06:03.845372592Z\"\n}",
```

Comment:
Maybe we should have some `event.*` fields in the parser. At least, `event.kind` fro example

Reply:
I will improve this part in another PR